### PR TITLE
Embedded_platforms

### PR DESCRIPTION
**Pull request by nathaniel-brough**

### CODE REVIEW

File: README.md
Comment (akirabaruah) [inline README.md]: Consider adding an "Approved" date column to track when the decision was finalized.

File: README.md
Comment (akirabaruah) [inline README.md]: What's the difference between a proposal that's a draft vs. under review? I'm wondering whether these sections are redundant.

File: README.md
Comment (akirabaruah) [reply README.md]: Also maybe helpful to list an "Approvers" column to clarify whose approval we need for a proposal to be approved.

File: README.md
Comment (nathaniel-brough) [reply README.md]: That's true, we can probably leave the drafts section out. I'm vaguely using [bazelbuild/proposals](https://github.com/bazelbuild/proposals) as a template. Though I think you are right in that it doesn't make much sense in this context.

File: README.md
Comment (akirabaruah) [inline README.md]: Authors(s) -> Author(s)

Extra "s"? Also in the table below.

**Review by akirabaruah — APPROVED** [review]

File: README.md
Comment (akirabaruah) [reply README.md]: Oh neat! Sounds good 👍 Maybe worth referencing that repo in this README along with the intended design process (assuming it will be similar)?

**Pull request merged**
